ArduCopter/control_auto.cpp

```diff
@@ -381,7 +381,9 @@ void Copter::auto_land_start(const Vector3f& destination)
     }
 
     // initialise yaw
-    set_auto_yaw_mode(AUTO_YAW_HOLD);
+    if (auto_yaw_mode != AUTO_YAW_LOOK_AT_HEADING) {
```

Comment:
I'm not very familiar with this code, but why only make this yaw mode the exception? For example, if it is in ROI mode wouldn't it make sense to keep looking at it while landing? The same probably applies to other yaw modes.